includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -149,4 +163,33 @@ function ( $payment_method ) use ( $available_payment_methods ) {
 		$this->wcpay_gateway->update_option( 'enabled_payment_method_ids', $payment_method_ids_to_enable );
 	}
 
+	/**
+	 * Updates WooCommerce Payments manual capture.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 */
+	private function update_is_manual_capture_enabled( WP_REST_Request $request ) {
+		if ( ! $request->has_param( 'is_manual_capture_enabled' ) ) {
+			return;
+		}
+
+		$is_manual_capture_enabled = $request->get_param( 'is_manual_capture_enabled' );
+
+		$this->wcpay_gateway->update_option( 'manual_capture', $is_manual_capture_enabled );
```

Comment:
It looks like this should have a value of `'yes'` or `'no'` (rather than boolean) in order to be accurately used:
- https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payment-gateway-wcpay.php#L1288
- https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payment-gateway-wcpay.php#L152

This is currently saving a boolean

```suggestion
		$this->wcpay_gateway->update_option( 'manual_capture', $is_manual_capture_enabled ? 'yes' : 'no' );
```